### DISCUSSION TITLE
Connecting Philips LWA017 (on support list)

### DISCUSSION THREAD

**Discussion by jasjnr** [Koenkk/zigbee2mqtt#10683] [Devices]
Hey,
Relatively new to this but am having difficulty connecting my LWA017 bulb. After resetting the bulb from the Philips BT app the device does connect to zigbee2mqtt but it says it's no supported. Here is the log entry from zigbee2mqtt:

Device '0x001788010b24329b' with Zigbee model 'LWA017' and manufacturer name 'Signify Netherlands B.V.' is NOT supported, please follow https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html

I have managed to connect a tradfri remote and motion sensor no problem but not the bulb. Running latest version on a CC2531 with attached aerial. The bulb is only 2m away from the CC2531 and does show up in the list of devices but I can't seem to do anything with it unfortunately.
Anyone else run into this? Open to suggestions.
TIA

Version | core-2021.12.8
-- | --
Installation Type | Home Assistant OS
Development | false
Supervisor | true
Docker | true
User | root
Virtual Environment | false
Python Version | 3.9.7
Operating System Family | Linux
Operating System Version | 5.10.63-v8
CPU Architecture | aarch64

Zigbee2mqtt
Current version: 1.18.1-1

**Comment by Koenkk** [comment]
1.18.1 is very old.

If you are still on 1.18.1 check first point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.19.0

**Comment by jasjnr** [comment]
Well thank you very much, that sorted it perfectly, good man!
